content/en/docs/distributions/aws/troubleshooting-aws.md

```diff
@@ -1,64 +1,62 @@
-
-### Kubeflow Uninstallation Failure
+If you see this error, you likely didn't install from a directory named `cluster_name` during setup. Please check `kubectl get configmaps aws-alb-ingress-controller-config -n kubeflow -o yaml` and make any needed change.
 
+Another reason could be that your subnets are not tagged so that Kubernetes knows which subnets to use for external load balancers. To fix this, ensure your cluster's public subnets are tagged with the **Key**: ```kubernetes.io/role/elb``` and **Value**: ```1```. See the [Amazon EKS User Guide](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html) for further details.
 
-```
-Error: couldn't delete KfApp:  (kubeflow.error): Code 500 with message: kfApp Delete failed for kustomize:  (kubeflow.error): Code 500 with message: error deleting kustomize manifests: [error evaluating kustomization manifest for knative: Timed out waiting for resource /knative-serving to be deleted. Error deleted resource is not cleaned up yet, error evaluating kustomization manifest for cert-manager: Timed out waiting for resource /cert-manager to be deleted. Error deleted resource is not cleaned up yet]
-Usage:
-  kfctl delete [flags]
-
-Flags:
-      --delete_storage   Set if you want to delete app's storage cluster used for mlpipeline.
-  -f, --file string      The local config file of KfDef.
-      --force-deletion   force-deletion output default is false
-  -h, --help             help for delete
-  -V, --verbose          verbose output default is false
-
-kfctl exited with error: couldn't delete KfApp:  (kubeflow.error): Code 500 with message: kfApp Delete failed for kustomize:  (kubeflow.error): Code 500 with message: error deleting kustomize manifests: [error evaluating kustomization manifest for knative: Timed out waiting for resource /knative-serving to be deleted. Error deleted resource is not cleaned up yet, error evaluating kustomization manifest for cert-manager: Timed out waiting for resource /cert-manager to be delete
+### Resource Not Found in `delete all`
 
+```shell
++ kubectl get ns/kubeflow
+Error from server (NotFound): namespaces "kubeflow" not found
++ kubectl get ns/kubeflow
+Error from server (NotFound): namespaces "kubeflow" not found
++ echo 'namespace kubeflow successfully deleted.'
 ```
 
-Due to a kustomize [issue](https://github.com/kubeflow/kfctl/issues/385), deletion will take up to 10 mins and it may show above errors. Don't worry about it, all resources have been deleted. You can build your kfctl with [change](https://github.com/kubeflow/kfctl/pull/386) to mitigate this issue.
-
+You can ignore any Kubernetes "resource not found" errors that occur during deletion.
 
 ### EKS Cluster Creation Failure
```

Comment:
In this case, should we directly refer them to EKS official trouble shooting page? 
https://docs.aws.amazon.com/eks/latest/userguide/troubleshooting.html

Reply:
I removed this entry based on comment above.